docs/detections/rules-ui-manage.asciidoc

```diff
@@ -72,6 +72,7 @@ Similarly, rules will be skipped if they can't be modified by a bulk edit. For e
 * Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
+** *Custom highlighted fields*: Add custom highlighted fields on all selected rules. You can choose any fields that are available in the <<update-sec-indices,default {elastic-sec} indices>>. To overwrite a rule's current set of custom highlighted fields, select the **Overwrite all selected rules' custom highlighted fields** option, then click **Save**. 
```

Comment:
What if a rule doesn't use the default Security indices? The instructions in rule creation say "You can choose any fields that are available in the indices you selected for the rule's data source," which might be something besides the default indices.

Reply:
Good question - @e40pud can fields from non-default Security indices be selected when bulk-updating custom highlighted fields for multiple rules? If it's not possible, it might be worth noting as a limitation here.

Reply:
For bulk changes, I'd assume maybe it'd be limited to just the indices that all the selected rules have in common, which may or may not be the default indices. But let's see what @e40pud says, thanks!

Reply:
In case of bulk action, we show all the fields from default security indices - specified in advanced settings. We do not take into account indices selected as rules' sources.

Even though some fields might be missing in drop down menu, user can always add those fields.

https://github.com/user-attachments/assets/59ee896f-1258-4e22-b02d-2784c25b9408

In this example, the field `some_field_1` does not exist in drop down menu, though I still can add it as a highlighted field. **NOTE**: this is only bulk action behaviour and does not happen on rule's editing page. There we know exactly the set of fields that available for the rule.

Reply:
We use default indices for bulk action to avoid performance issues which can happen when user selects multiple rules with lots of different indices, meaning we would need to fetch lots of fields. And allowing users to add any field as a custom option during bulk action removes the limitation of the fields that can be added as highlighted fields. Hope that makes sense - happy to chat over zoom about it if needed.

Reply:
Makes perfect sense — thanks for the helpful explanation @e40pud! 

@nastasha-solomon How about something like this:

```suggestion
** *Custom highlighted fields*: Add custom highlighted fields on all selected rules. You can choose any fields that are available in the <<update-sec-indices,default {elastic-sec} indices>>, or enter field names from other indices. To overwrite a rule's current set of custom highlighted fields, select the **Overwrite all selected rules' custom highlighted fields** option, then click **Save**. 
```

Reply:
Works for me @joepeeples, and thanks both for the input!